### PR TITLE
[CLC-67] Updated table printer

### PR DESCRIPTION
**Pull request by yuce**
* Updated table to have multiline rows.
* Non-stream tables observe `CLC_MAX_COLS` environment variable.
* Other refactoring.

### CODE REVIEW

File: base/printers.go [hunk 1]
```diff
@@ -55,17 +55,24 @@ type TablePrinter struct{}
 func (pr *TablePrinter) PrintStream(ctx context.Context, w io.Writer, rp output.RowProducer) error {
 	mc, _ := consolesize.GetConsoleSize()
 	if mc <= 0 {
```
Comment (kutluhanmetin) [inline base/printers.go:57]: it cannot be lower than zero that GetConsoleSize returns uint16 for both return values.

File: internal/table/table.go [hunk 2]
```diff
@@ -137,3 +164,35 @@ func withColor(c *color.Color, f func()) {
...
 	color.Unset()
 }
+
+func splitLines(text string, maxWidth int) lines {
+	// just a precaution to not allocate the universe
+	if maxWidth > 65535 {
+		maxWidth = 65535
+	}
+	if maxWidth == 0 {
+		panic("splitLines: maxWidth cannot be 0")
```
Comment (kutluhanmetin) [inline internal/table/table.go:174]: Should we panic?

**Review by kutluhanmetin — COMMENTED** [review]
Just 2 minor comments

File: base/printers.go (hunk 1 above)
Comment (yuce) [reply base/printers.go:57]: > At least we can reduce them to a single if block: if mc == 0
``
	if mc <= 0 {
		mc = tableMaxcols()
	}
	if mc < 1 {
		mc = 1000
	}
```
`TableMaxCols` may also return `0`, so need to check the value of `mc` again.

File: internal/table/table.go (hunk 2 above)
Comment (yuce) [reply internal/table/table.go:174]: `maxWidth` should never be `0` in the normal operation, that would be an exceptional case.

**Review by kutluhanmetin — APPROVED** [review]
LGTM

**Pull request merged**